.github/workflows/integration-test.yml

```diff
@@ -9,11 +9,7 @@ on:
 env:
   GO_VERSION: "~1.19"
   # renovate: datasource=github-releases depName=cert-manager/cert-manager
-  CERT_MANAGER_VERSION: "v1.10.0"
-  # renovate: datasource=github-tags depName=kudobuilder/kuttl
```

Comment:
The renovate comment for kuttl should stay here :)

Reply:
ups, deleted the wrong one :sweat_smile: